parcels/particle.py

```diff
@@ -257,7 +314,8 @@ def remove(self, indices):
         return particles
 
     def execute(self, pyfunc=AdvectionRK4, starttime=None, endtime=None, dt=1.,
-                runtime=None, output_file=None, output_interval=-1, show_movie=False):
+                runtime=None, interval=None, output_file=None, tol=None,
+                show_movie=False):
```

Comment:
I don't think that the keyword `interval` is specific enough here. Users who don't go into the code won't understand the difference with the `dt` keyword.
I think `output_interval` worked better. Yes, the keyword does more than control output, but to a user, that's what it's about